content-services/latest/develop/sdk.md

```diff
@@ -206,7 +203,7 @@ Alfresco recommends that you keep up-to-date with all the Docker releases. If yo
     14: 4.2.0
     15: 4.3.0
     15: 4.4.0
-    16: 4.5.0   
+    16: 4.6.0   
```

Comment:
Usually the previous version remains in this list - you can check it if you follow the "Getting started" steps.

Reply:
Has been done in https://github.com/Alfresco/docs-alfresco/pull/1074